dotcom-rendering/src/web/components/TopRightAdSlot.importable.tsx

```diff
@@ -37,50 +39,74 @@ export const TopRightAdSlot = ({
 	format?: ArticleFormat;
 }) => {
 	const adBlockerDetected = useAdBlockInUse();
-	const isSignedIn =
-		!isServer && !!getCookie({ name: 'GU_U', shouldMemoize: true });
+	const [isShady, setIsShady] = useState(false);
 
-	if (
-		adBlockerDetected &&
-		!isSignedIn &&
-		!shouldHideReaderRevenue &&
-		!isPaidContent &&
-		!isServer
-	) {
-		return <ShadyPie format={format} editionId={editionId} />;
-	}
+	const ABTestAPI = useAB();
+
+	const userInShadyPieTestVariant =
+		ABTestAPI?.isUserInVariant('shadyPieClickThrough', 'variant') &&
+		format?.theme == ArticlePillar.Lifestyle;
 
-	// Otherwise return the classic ad slot
-	return (
-		<div
-			id="top-right-ad-slot"
-			css={css`
-				position: static;
-				height: ${MOSTVIEWED_STICKY_HEIGHT}px;
-			`}
-		>
+	useEffect(() => {
+		const isSignedIn =
+			!isServer && !!getCookie({ name: 'GU_U', shouldMemoize: true });
+		setIsShady(
+			(adBlockerDetected &&
+				!isSignedIn &&
+				!shouldHideReaderRevenue &&
+				!isPaidContent &&
+				!isServer) ||
+				!!userInShadyPieTestVariant,
```

Comment:
Wondering if we might want to adjust these rules slightly so that even if you're in the variant you won't see shady pie if you're signed in, it's paid content or reader-revenue should be hidden?

```suggestion
			(adBlockerDetected ||
				!!userInShadyPieTestVariant) &&
				!isSignedIn &&
				!shouldHideReaderRevenue &&
				!isPaidContent,
```

Maybe I've misunderstood the setup of this test though so ignore me if this is the intended behaviour!

Reply:
Good catch! That sounds sensible